packages/devtools_app/lib/src/screens/memory/panes/diff/widgets/class_details.dart

```diff
@@ -137,69 +127,39 @@ class _RetainedSizeColumn extends ColumnData<StatsByPathEntry> {
       )!;
 }
 
-class _RetainingPathTable extends StatefulWidget {
+class _RetainingPathTable extends StatelessWidget {
   _RetainingPathTable({
     Key? key,
     required this.data,
-    required this.sorting,
     required this.controller,
     required this.item,
-  })  : _columns = _tableColumns(sorting),
-        super(key: key);
-
-  final ClassStats data;
-  final ColumnSorting sorting;
-  final SnapshotInstanceItem item;
-  final DiffPaneController controller;
-
-  final List<ColumnData<StatsByPathEntry>> _columns;
-
-  static List<ColumnData<StatsByPathEntry>> _tableColumns(
-    ColumnSorting sorting,
-  ) {
-    final _shallowSizeColumn = _ShallowSizeColumn();
-
-    final result = <ColumnData<StatsByPathEntry>>[
+  }) : super(key: key) {
+    _columns = <ColumnData<StatsByPathEntry>>[
       _RetainingPathColumn(),
       _InstanceColumn(),
       _shallowSizeColumn,
       _RetainedSizeColumn(),
     ];
-
-    if (!sorting.initialized) {
-      sorting
-        ..direction = SortDirection.descending
-        ..columnIndex = result.indexOf(_shallowSizeColumn)
-        ..initialized = true;
-    }
-
-    return result;
   }
 
-  @override
-  State<_RetainingPathTable> createState() => _RetainingPathTableState();
-}
+  final ClassStats data;
+  final SnapshotInstanceItem item;
+  final DiffPaneController controller;
+
+  late final List<ColumnData<StatsByPathEntry>> _columns;
+  final _shallowSizeColumn = _ShallowSizeColumn();
```

Comment:
static const

Reply:
it cannot be const because of font size scaling